protostar/starknet_gateway/network_config.py

```diff
@@ -40,32 +47,36 @@ def build(
 
     @staticmethod
     def get_starknet_networks() -> List[str]:
-        return list(NETWORKS.keys())
+        return KNOWN_NETWORKS
 
     @classmethod
-    def from_starknet_network_name(
-        cls,
-        starkware_network_name: str,
-    ) -> "NetworkConfig":
-        if starkware_network_name not in NETWORKS:
+    def from_starknet_network_name(cls, starkware_network_name: str) -> "NetworkConfig":
+        if starkware_network_name not in NetworkConfig.get_starknet_networks():
             raise UnknownStarkwareNetworkException()
 
         contract_explorer_search_url_mapping = {
-            "alpha-goerli": "https://goerli.voyager.online/contract",
-            "alpha-mainnet": "https://voyager.online/contract",
+            TESTNET: "https://goerli.voyager.online/contract",
+            MAINNET: "https://voyager.online/contract",
```

Comment:
Starknet CLI uses `alpha-goerli` etc.

At the other hand, I can never memorize these names, so maybe it's better to change them.

Reply:
in my mind it's better to keep things similar to starknet.py

Reply:
didn't @Radinyn write related functions somewhere in deploy cheatcode impl?

Reply:
Maybe we should support both naming styles and deprecate the StaknetCLI's names in the future versions.

Reply:
I kind of don't understand why we would do that, do we have a version released which does anything with networks?

Reply:
The `deploy` command was released in version 0.2.1, and it supported network name similarly to Starknet CLI.

Reply:
done